src/main/kotlin/com/terraformation/backend/customer/db/ProjectStore.kt

```diff
@@ -182,3 +201,17 @@ class ProjectStore(
     }
   }
 }
+
+data class ProjectCohortData(
+    val cohortId: CohortId,
+    val cohortPhase: CohortPhase,
+) {
+  companion object {
+    fun of(record: Record): ProjectCohortData {
+      return ProjectCohortData(
+          cohortId = record[COHORTS.ID]!!,
+          cohortPhase = record[COHORTS.PHASE_ID]!!,
+      )
+    }
+  }
+}
```

Comment:
Maybe move this to a separate file in the `model` package.

Reply:
When is it okay and not? I see `data class`es in other non-model packages.

Reply:
> When is it okay and not? I see data classes in other non-model packages.

Fair question, and I'm probably inconsistent about it myself. I guess the pattern the code _mostly_ follows -- and I'd be totally down with fixing this where it's not true and writing it down as a style rule if people agree -- is:

* If a class is only used within a single source file, never passed to code in another file, it's fine to colocate. Example: the payload classes for controllers.
* If there are a number of similar classes and they're all reasonably small, they can live together. Example: the exception classes in the various `Exceptions.kt` files.

Reply:
I guess a third rule, or an exception, would be that the first two rules are relaxed for tests.

Reply:
Excellent, I think those rules make a lot of sense, thanks!